DOCS/man/options.rst

```diff
@@ -4876,6 +4876,69 @@ DVB
     An example ``input.conf`` could contain:
     ``H cycle dvbin-channel-switch-offset up``, ``K cycle dvbin-channel-switch-offset down``
 
+DVBV5
+-----
+
+``--dvbv5-adapter=<0-100>``
+    Use the given adapter. Default value: 0.
+
+``--dvbv5-audio_pid=<0-9999>``
+    Select a different audio Packet ID (PID). The default is to use
+    the first audio PID found at the channel-name-file.
+
+``--dvbv5-cc=<country code>``
+    Set the default country to be used by the MPEG-TS parsers, in
+    ISO 3166-1 two letter code. If not specified, the default
+    charset is guessed from the locale environment variables.
+
+``--dvbv5-lna=<-1-1>``
+    Enable, disable or put LNA power in auto mode. Not all frontends
+    support it. Valid values are:
+
+    0 - disable
+
+    1 - enable
+
+    -1 - auto
```

Comment:
This should be a `no/yes/auto` option instead of misusing integers

Reply:
Good idea! Fixed in [force-push](https://github.com/mpv-player/mpv/compare/9bffd58c5171b34ac972795c4e96fe2cf6a17cbb..57ea68124fa2cd0a05761215a6714cdeb4a278dc)